nix/haskell.nix

```diff
@@ -14,6 +14,7 @@ let
   r-packages = with pkgs.rPackages; [pkgs.R tidyverse dplyr stringr MASS];
   agdaWithStdlib = agdaPackages.agda.withPackages [ agdaPackages.standard-library ];
   project = haskell-nix.stackProject' {
+    compiler-nix-name = "ghc8102-experimental";
```

Comment:
Let's have a comment explaining what this is, for other people.

Reply:
Comment added `Use ghc 8.10.2 with the experimental core interface files patch`